tools/benchmark_tool/openvino/tools/benchmark/main.py

```diff
@@ -170,9 +170,9 @@ def get_device_type_from_name(name) :
 
             ## high-level performance modes
             if properties.hint.performance_mode() not in config[device].keys():
-                config[device][properties.hint.performance_mode()] = get_performance_hint(device)
-
-            perf_hint = config[device][properties.hint.performance_mode()]
+                perf_hint = get_performance_hint(device)
+                if perf_hint == properties.hint.PerformanceMode.LATENCY and perf_hint == properties.hint.PerformanceMode.THROUGHPUT and perf_hint == properties.hint.PerformanceMode.CUMULATIVE_THROUGHPUT:
```

Comment:
`perf_hint` can't be3 equal to every PerformanceMode. It should be `or` here

Reply:
my bad. updated.